packages/base/src/dialogs/symbology/vector_layer/VectorRendering.tsx

```diff
@@ -36,7 +37,13 @@ const VectorRendering = ({
     setSelectedRenderType(renderType);
 
     const options: Record<string, string[]> = {
-      VectorLayer: ['Single Symbol', 'Graduated', 'Categorized', 'Heatmap'],
+      VectorLayer: [
+        'Single Symbol',
+        'Graduated',
+        'Categorized',
+        'Canonical',
+        'Heatmap'
+      ],
```

Comment:
I'd put Canonical here on a condition if the layer has some color attribute

Reply:
Good call!

Reply:
Implemented this logic for `Graduated` and `Categorized` (require >=1 numeric field) and `Canonical` (require >=1 hex code field)

Reply:
Thanks, this is much better now :)